java/bench/spark/pom.xml

```diff
@@ -171,8 +171,9 @@
                     <exclude>META-INF/NOTICE-binary</exclude>
                     <exclude>META-INF/NOTICE.md</exclude>
                     <exclude>META-INF/NOTICE.txt</exclude>
-                    <exclude>META-INF/DUMMY.SF</exclude>
-                    <exclude>META-INF/DUMMY.DSA</exclude>
+                    <exclude>META-INF/*.SF</exclude>
+                    <exclude>META-INF/*.DSA</exclude>
+                    <exclude>META-INF/*.RSA</exclude>
```

Comment:
```
[WARNING] eclipse-collections-11.1.0.jar, eclipse-collections-api-11.1.0.jar define 4 overlapping resources:
[WARNING]   - LICENSE-EDL-1.0.txt
[WARNING]   - LICENSE-EPL-1.0.txt
[WARNING]   - META-INF/ECLIPSE_.RSA
[WARNING]   - META-INF/ECLIPSE_.SF
```

<img width="473" alt="image" src="https://github.com/apache/orc/assets/3898450/79eaf13b-9cca-4254-a5bf-650f7b5016d0">

Reply:
Currently, Spark has fixed this problem by upgrading the `arrow-vector` version, so there is no modification here.

 [SPARK-47981][BUILD] Upgrade `Arrow` to 16.0.0